hack/kube/components/dev/grafana-alloy.yaml

```diff
@@ -1,25 +1,26 @@
-# Generated with `helm template grafana/grafana-agent --set=agent.mode=flow --set=controller.type=statefulset` and modified.
+# Generated with `helm template grafana/alloy --name-template=grafana-alloy --set=controller.type=statefulset --set=configReloader.enabled=false --set=crds.create=false`.
 ---
-# Source: grafana-agent/templates/serviceaccount.yaml
+# Source: alloy/templates/serviceaccount.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: grafana-agent
+  name: grafana-alloy
+  namespace: enduro-sdps
   labels:
-    app.kubernetes.io/name: grafana-agent
+    app.kubernetes.io/name: alloy
```

Comment:
Not sure is this even shown/used but, should we match this label with the metadata name above? There are many repetitions below, also for the prometheus-server.

Reply:
Yeah good point, I wasn't really paying attention to that, thanks.

Reply:
```
$ kubectl get all -A -l app.kubernetes.io/name=grafana-alloy
NAMESPACE     NAME                  READY   STATUS    RESTARTS   AGE
enduro-sdps   pod/grafana-alloy-0   1/1     Running   0          28m

NAMESPACE     NAME                    TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                       AGE
enduro-sdps   service/grafana-alloy   ClusterIP   10.43.183.218   <none>        12345/TCP,4317/TCP,4318/TCP   28m

NAMESPACE     NAME                             READY   AGE
enduro-sdps   statefulset.apps/grafana-alloy   1/1     28m
```